ColabTool/Shared/PostFooter/index.jsx

```diff
@@ -78,7 +78,15 @@ const renderEdit = ({
   );
 };
 
-const renderIcon = isPastDue => (isPastDue ? <WarningIcon color={'torchRed'} /> : <ClockIcon />);
+const renderIcon = ({ isPastDue, scheduledAt }) => {
```

Comment:
There's an ambiguous state that makes it a little trickier to understand here. Returning null is a little clearer here I think :smile:
 
```js
const renderIcon = ({ isPastDue, scheduledAt }) => {
   if (isPastDue) {
     return (<WarningIcon color={'torchRed'} />);
   } else if (scheduledAt) {
     return (<ClockIcon />);
   }
   return null;
 };
```